docs/admin/troubleshooting/cluster-health-check.md

```diff
@@ -1,8 +1,82 @@
-# Verifying Cluster Health
+# Cluster Health and Troubleshooting
 
-Following is a set of tests that determine the Run:ai cluster health:
+## Cluster is Disconnected
 
-## Verify that data is sent to the cloud
+When a cluster is in the status “Disconnected”, this means that no communication from the Run:ai cluster services reaches the Run:ai Control Plane. 
+
+This may reflect a networking issue from or to your Kubernetes cluster regardless of Run:ai components. In some cases, it may indicate an issue with one or more Run:ai services that communicate with the Control Plane, which are:
```

Comment:
```suggestion
This may reflect a networking issue from or to your Kubernetes cluster regardless of Run:ai components. In some cases, it may indicate an issue with one or more Run:ai services that communicate with the Control Plane. These are:
```